tests/config/test_data.config

```diff
@@ -74,12 +74,14 @@ params {
                 scaffolds_fasta                                = "${test_data_dir}/genomics/sarscov2/illumina/fasta/scaffolds.fasta"
 
                 assembly_gfa                                   = "${test_data_dir}/genomics/sarscov2/illumina/gfa/assembly.gfa"
+
+                picard_readlist_test_single_end_bam            = "${test_data_dir}/genomics/sarscov2/illumina/picard/test.single_end.bam.readlist.txt"
```

Comment:
```suggestion
                assembly_gfa                                   = "${test_data_dir}/genomics/sarscov2/illumina/gfa/assembly.gfa"

                test_single_end_bam_readlist_txt               = "${test_data_dir}/genomics/sarscov2/illumina/picard/test.single_end.bam.readlist.txt"
```

Reply:
The name is always the same as the final file but dots replaced with underscores